src/com/facebook/buck/swift/SwiftLibraryDescription.java

```diff
@@ -530,4 +554,32 @@ public static boolean isSwiftTarget(BuildTarget buildTarget) {
      */
     Optional<String> getTargetSdkVersion();
   }
+
+  private static class FrameworkLinkerArgs extends FrameworkPathArg {
+    @AddToRuleKey final AddsToRuleKeyFunction<FrameworkPath, Path> frameworkPathToSearchPath;
+
+    public FrameworkLinkerArgs(
```

Comment:
@xianwen: We do not need this anymore if we just pass the list of frameworks + function.

Reply:
No, we don't.
I'll remove this and also update a few other places accordingly.